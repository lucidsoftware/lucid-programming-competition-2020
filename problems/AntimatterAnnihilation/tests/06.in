some_problems_antimatterare_hard_some_are_easy_not_suantimatterre_antimatterwhich_thisantantimatterimatterantimatter_is_xxantimaantimattertterxxxxantimattantimatterer