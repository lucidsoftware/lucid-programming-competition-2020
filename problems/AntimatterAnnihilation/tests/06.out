some_problems_are_hard_some_are_easy_not_sure_which_this_is_xxxxxx